packages/runtime/src/helpers/edge.ts

```diff
@@ -200,7 +200,18 @@ export const writeEdgeFunctions = async (netlifyConfig: NetlifyConfig) => {
   const edgeFunctionRoot = resolve('.netlify', 'edge-functions')
   await emptyDir(edgeFunctionRoot)
 
-  if (!process.env.NEXT_DISABLE_EDGE_IMAGES) {
+  const { publish } = netlifyConfig.build
+  const nextConfigFile = await getRequiredServerFiles(publish)
+  const nextConfig = nextConfigFile.config
+  await copy(getEdgeTemplatePath('../edge-shared'), join(edgeFunctionRoot, 'edge-shared'))
+  await writeJSON(join(edgeFunctionRoot, 'edge-shared', 'nextConfig.json'), nextConfig)
+
+  if (
+    process.env.NEXT_DISABLE_EDGE_IMAGES !== '1' &&
+    process.env.NEXT_DISABLE_EDGE_IMAGES !== 'true' &&
+    process.env.NEXT_DISABLE_NETLIFY_EDGE !== 'true' &&
+    process.env.NEXT_DISABLE_NETLIFY_EDGE !== '1'
```

Comment:
[sand] It's a shame I wasn't able to get my IPX work in before these changes because I'm adding the `isEnvSet` method that we also have in the [Gatsby plugin](https://github.com/netlify/next-runtime/blob/276b948c288e3b592e4db6a96c206dd61c3d9b51/packages/runtime/src/helpers/utils.ts#L232) to help make logic like this a little easier on the eyes.

You're free to include the method in your PR if you'd like to use here and I can resolve the conflicts in my branch, but no worries if you're a bit pressed for time and would rather leave this as is for the time being.

Reply:
@ericapisani I've updated it to use [destr](https://github.com/unjs/destr)

Reply:
Yeah, but bear in mind we can't use it inside the patched code